FAQ.md

```diff
@@ -36,9 +36,12 @@ to integrate with container ecosystems.
 
 ### What processors does Firecracker support?
 
-The Firecracker VMM is built to be processor agnostic. Intel processors are
-supported for production workloads. Support for AMD and Arm processors is in
-developer preview.
+The Firecracker VMM is built to be processor agnostic. Intel (Skylake and Cascade
```

Comment:
The answer here should be the same as in this discussion https://github.com/firecracker-microvm/firecracker/pull/2194. I'll submit only the i3 to m5 change and let that discussion to finish and add this part to that commit

Reply:
As @sandreim suggested, I think we could simply link here the `Supported platforms` heading from `Readme.md`. WDYT? This way, we wouldn't have to duplicate that paragraph and would only have one place to maintain it.

Reply:
I will drop the commit modifying the FAQ and wait until #2194 is merged.